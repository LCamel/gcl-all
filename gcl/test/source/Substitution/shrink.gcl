var global1 : Int 
var global2 : Int 
{: 
    P bound = global1 = bound
:}

global1 := global1 + 1
global2 := global2 + 1

{ P 3 }
