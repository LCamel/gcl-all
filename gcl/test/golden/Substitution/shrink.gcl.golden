1: P 3
  P 3 ===> global1 = 3
    
2: P [ global1 \ global1 + 1 ] 3
  P [ global1 \ global1 + 1 ] 3 ===> global1 + 1 = 3
    
3: P [ global1 \ global1 + 1 ] 3
  P [ global1 \ global1 + 1 ] 3 ===> global1 + 1 = 3
    